src/components/HelpScoutBeacon/HelpScoutBeacon.js

```diff
@@ -0,0 +1,377 @@
+      )}
+      <ToggleDialogueButton
+        open={mode === OPENED || mode === OPENING}
+        onToggle={handleToggle}
+      />
+    </React.Fragment>
+  )
+})
+
+HelpOptIn.propTypes = {
+  onOptIn: PropTypes.func.isRequired,
+  optedIn: PropTypes.bool.isRequired,
+}
+
+const ToggleDialogueButton = ({ open, onToggle }) => {
+  return (
+    <RoundButton
+      onClick={onToggle}
+      css={`
+        margin-left: calc(100% - 60px);
+        margin-top: ${2 * GU}px;
+      `}
+    >
+      <Transition
+        native
+        items={open}
+        from={{ opacity: 0, enterProgress: 0 }}
+        enter={{ opacity: 1, enterProgress: 1 }}
+        leave={{ opacity: 0, enterProgress: 1 }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          /* eslint-disable react/prop-types */
+          (({ opacity, enterProgress }) => (
+            <AnimatedDiv
+              style={{
+                opacity,
+                transform: enterProgress.interpolate(
+                  v => `rotate(${45 * (1 - v)}deg)`
+                ),
+              }}
+            >
+              <IconClose
+                color={theme.gradientText}
+                css={`
+                  width: auto;
+                  height: 22px;
+
+                  & path {
+                    fill: ${theme.gradientText};
+                    transform: scale(2.2);
+                    opacity: 1;
+                  }
+                `}
+              />
+            </AnimatedDiv>
+          ))
+        }
+      </Transition>
+      <Transition
+        native
+        items={!open}
+        from={{ opacity: 0, enterProgress: 0 }}
+        enter={{ opacity: 1, enterProgress: 1 }}
+        leave={{ opacity: 0, enterProgress: 1 }}
+        config={springs.smooth}
+      >
+        {show =>
+          show &&
+          /* eslint-disable react/prop-types */
+          (({ opacity, enterProgress }) => (
+            <AnimatedDiv
+              style={{
+                opacity,
+                transform: enterProgress.interpolate(
+                  v => `
+                    scale3d(${1 - (1 - v) * 0.03}, ${1 - (1 - v) * 0.03}, 1)
+                  `
+                ),
+              }}
+            >
+              <IconQuestion />
+            </AnimatedDiv>
+          ))
+        }
+      </Transition>
+    </RoundButton>
+  )
+}
+
+ToggleDialogueButton.propTypes = {
+  open: PropTypes.bool.isRequired,
+  onToggle: PropTypes.func.isRequired,
+}
+
+const OptInDialogue = React.memo(({ onClose, onOptIn, ...styles }) => {
+  return (
+    <animated.div {...styles}>
+      <Wrapper>
+        <Header>
+          <h2
+            css={`
+              font-size: 18px;
+              line-height: 22px;
+            `}
+          >
+            How can we help?
+          </h2>
+          <img
+            css={`
+              margin-top: ${2.5 * GU}px;
+            `}
+            src={logo}
+            alt="Aragon logo"
```

Comment:
In this context, it’s better to not describe the image since it’s more decorative than part of the textual content (so that it doesn’t read “How can we help? − Aragon logo − We need our consent”).

```suggestion
            alt=""
```